docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -1,86 +1,90 @@
-        return RestClient.getExpectJSON<ITodo[]>(session, Typicode.TODO_URI);
-    }
+        public static readonly TODO_URI = "/todos";
 
-    public static getTodo(session: AbstractSession, id: number): Promise<ITodo> {
-        Logger.getAppLogger().trace("Typicode.getTodos() called with id " + id);
-        ImperativeExpect.toNotBeNullOrUndefined(id, "id must be provided");
-        const resource = Typicode.TODO_URI + "/" + id;
-        return RestClient.getExpectJSON<ITodo>(session, resource);
+        public static getTodos(session: AbstractSession): Promise<ITodo[]> {
+            Logger.getAppLogger().trace("Typicode.getTodos() called");
+            return RestClient.getExpectJSON<ITodo[]>(session, Typicode.TODO_URI);
+        }
+
+        public static getTodo(session: AbstractSession, id: number): Promise<ITodo> {
+            Logger.getAppLogger().trace("Typicode.getTodos() called with id " + id);
+            ImperativeExpect.toNotBeNullOrUndefined(id, "id must be provided");
+            const resource = Typicode.TODO_URI + "/" + id;
+            return RestClient.getExpectJSON<ITodo>(session, resource);
+        }
     }
-}
 
-```
+    ```
+
+    The `Typicode` class provides two programmatic APIs, `getTodos` and `getTodo`, to get an array of `ITodo` objects or a specific `ITodo`, respectively.
 
-The `Typicode` class provides two programmatic APIs, `getTodos` and `getTodo`, to get an array of `ITodo` objects or a specific
-`ITodo` respectively. The Node.js APIs use `@zowe/imperative` infrastructure to provide logging, parameter validation,
-and to call a REST API. See the [Imperative CLI Framework documentation](https://github.com/zowe/imperative/wiki) for more information.
+    The Node.js APIs use `@zowe/imperative` infrastructure to provide logging, parameter validation, and to call a REST API. See the [Imperative CLI Framework documentation](https://github.com/zowe/zowe-cli/wiki) for more information.
 
-#### Exporting interface and programmatic API for other Node.js applications
+#### Exporting the interface and programmatic API for other Node.js applications
 
-Update [zowe-cli-sample-plugin/src/index.ts](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/src/index.ts) to contain the following:
+Edit the [zowe-cli-sample-plugin/src/index.ts](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/src/index.ts) file to contain the following code:
 
 ```typescript
 export * from "./api/doc/ITodo";
 export * from "./api/Typicode";
 ```
 
-A sample invocation of your API might look similar to the following, if it were used by a separate, standalone Node.js application:
+This allows a separate, standalone Node.js application to use APIs from the sample Typicode plug-in to get data from the REST API at jsonplaceholder.typicode.com.
+
+The following code is an example of how a Node.js application could apply your API: **[re: Lines 84-86: I did some editing. Is this correct?]**
```

Comment:
@adam-wolfe please respond

Reply:
Line 84 looks good. I might change line 86 to something like:

"The following code is an example of how a Node.js application could import classes from your API to interact with the Typicode REST API:"